Couldn't compile winmd_macros

I am trying to compile the minesweeper example from the [blog post](https://blogs.windows.com/windowsdeveloper/2020/04/30/rust-winrt-public-preview/), and am getting the following error.

>error[E0432]: unresolved import \`proc_macro\`
> --> C:\Users\aw14s\.cargo\git\checkouts\winrt-rs-5b24704ba4b09dbb\837e0cb\crates\winmd\macros\src\lib.rs:1:5   
>  |
>1 | use proc_macro::TokenStream;
>  |     ^^^^^^^^^^ help: a similar path exists: \`syn::proc_macro\`
>
>error: aborting due to previous error
>
>For more information about this error, try \`rustc --explain E0432\`.
>error: could not compile \`winmd_macros\`.

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -19,3 +19,17 @@ jobs:
 
     - name: fmt
       run: cargo fmt --all -- --check
+
+  min-supported:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: install
+      run: rustup install 1.40.0
+
+    - name: check
+      run: cargo +1.40.0 check --all --bins --examples
+
+    - name: tests
+      run: cargo +1.40.0 test --all 

--- a/crates/macros/src/lib.rs
+++ b/crates/macros/src/lib.rs
@@ -1,3 +1,5 @@
+extern crate proc_macro;
+
 use proc_macro::{TokenStream, TokenTree};
 use winmd::{TypeLimits, TypeReader, TypeStage};
 

--- a/crates/winmd/macros/src/lib.rs
+++ b/crates/winmd/macros/src/lib.rs
@@ -1,3 +1,5 @@
+extern crate proc_macro;
+
 use proc_macro::TokenStream;
 use proc_macro2::TokenStream as TokenStream2;
 use quote::{format_ident, quote};

--- a/crates/winmd/src/types/enum.rs
+++ b/crates/winmd/src/types/enum.rs
@@ -88,8 +88,9 @@ impl Enum {
 }
 
 fn bitwise_operators(name: &TokenStream, value_type: &EnumConstant) -> TokenStream {
-    if matches!(value_type, EnumConstant::I32(_)) {
-        return quote! {};
+    match value_type {
+        EnumConstant::I32(_) => return quote! {},
+        _ => {}
     }
 
     quote! {

--- a/crates/winmd/src/types/required_interfaces.rs
+++ b/crates/winmd/src/types/required_interfaces.rs
@@ -30,8 +30,9 @@ fn kind(reader: &TypeReader, required: InterfaceImpl) -> InterfaceKind {
     for attribute in required.attributes(reader) {
         let name = attribute.name(reader);
 
-        if matches!(name, ("Windows.Foundation.Metadata", "DefaultAttribute")) {
-            return InterfaceKind::Default;
+        match name {
+            ("Windows.Foundation.Metadata", "DefaultAttribute") => return InterfaceKind::Default,
+            _ => {}
         }
     }
 

